build/lib/internal.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// Internal build state for `build_resolvers`, `build_runner_core` and
```

Comment:
Any particular reason the two exported files aren't just imported where you need it, or that this isn't inside "src" or something, to discourage more than the documentation on "library"?